.chloggen/revert_resource.yaml

```diff
@@ -0,0 +1,22 @@
+# Use this changelog template to create an entry for release notes.
+#
+# If your change doesn't affect end users you should instead start
+# your pull request title with [chore] or use the "Skip Changelog" label.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: breaking
```

Comment:
Is this really breaking? If not, I think we can probably omit the changelog for this?

Reply:
This temp  was published in v1.24. That was the reason why I have added a changelog entry. Do you think we can just skip changelog in this case?

Reply:
I'd say so, yeah. The temp fix was only because it broke the code generation in the build tools. But AFAIK now the build tools can also generate the code in other ways, so I think reverting the registry here is a non breaking change.